src/SchemaReflection/Join.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+declare(strict_types=1);
+
+namespace staabm\PHPStanDba\SchemaReflection;
+
+/**
+ * @api
+ */
+final class Join
```

Comment:
not sure it makes sense to model joins here. maybe we should instead use `SqlFtw\Sql\Dml\TableReference\Join` instead, as we need the low level information